org.eclipse.jdt.ls.core/pom.xml

```diff
@@ -77,42 +77,24 @@
 									String wrapperChecksumUrl;
 									String sha256
 								}
-								HttpClient client = HttpClient.newHttpClient()
+
+								HttpClient client = HttpClient.newBuilder()
+									.followRedirects(Redirect.NORMAL)
+									.build();
 								def futures = []
 								versions.each {
-									if (it.wrapperChecksumUrl == null) {
+									if (it.wrapperChecksumUrl == null || it.nightly || it.snapshot || it.rcFor != "" || it.broken) {
 										return
 									}
 									HttpRequest request = HttpRequest.newBuilder().uri(URI.create(it.wrapperChecksumUrl)).build()
-									futures.add(client.sendAsync(request, BodyHandlers.ofString()).thenApply { response ->
-										if (response.statusCode() == 301) {
```

Comment:
Are you eliminating redirection support because none of the links currently require it ? Wouldn't it be good to have this just in case ?

Reply:
The client does that implicitly if configured to:

```
HttpClient client = HttpClient.newBuilder()
	.followRedirects(Redirect.NORMAL)
	.build();
```